astroid/bases.py

```diff
@@ -71,7 +71,13 @@ def _is_property(meth):
 
 
 class Proxy(object):
-    """a simple proxy object"""
+    """a simple proxy object
+
+    Note:
+
+    Subclasses of this object will need a custom __getattr__
```

Comment:
I wonder if we should try https://github.com/PyCQA/astroid/issues/205 to get rid of this `Proxy` implementation instead.